packages/cqrs/pubspec.yaml

```diff
@@ -7,10 +7,11 @@ description: >-
   queries and commands.
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
In PR #40 the quotes were changed the other way around in `leancode_lint` pubspec. We have to decide on some convention for it

Reply:
ugh, autoformatter.

sorry

But TBH I don't know why are we forcing single quotes when the autoformatter itself uses double quotes. [some tiny research here](https://github.com/leancodepl/flutter_corelibrary/pull/40#discussion_r755114223)